data/en/features.yml

```diff
@@ -6,6 +6,14 @@
 corefeatures:
   title: "Our core features"
   columns:
+    - title: "Web-based"
+      icon: "fa fa-firefox"
+      content: "Your users don't need extra software. HedgeDoc runs in their browser."
```

Comment:
```suggestion
      content: "You don't need extra software. HedgeDoc runs in the browser."
```

Reply:
I also thought about this but that sounds a bit like "You don't need to install anything. Not even the software itself! Just use your browser and it magically works!". But on the other hand the next point is "HedgeDoc runs on your server." Sooo I'm not sure.

Reply:
(I changed it for now)

Reply:
How about "The editor runs in the browser" "The markdown editor runs in the browser" ?

Reply:
Changed it to "The editor runs in the browser"